project/plugins.sbt

```diff
@@ -1,6 +1,6 @@
-addSbtPlugin("org.scalameta" % "sbt-scalafmt" % "2.0.6")
-addSbtPlugin("com.typesafe.sbt" % "sbt-native-packager" % "1.6.1")
-addSbtPlugin("com.github.cb372" % "sbt-explicit-dependencies" % "0.2.12")
-addSbtPlugin("org.scoverage" % "sbt-scoverage" % "1.6.1")
+addSbtPlugin("org.scalameta" % "sbt-scalafmt" % "2.4.2")
+addSbtPlugin("com.typesafe.sbt" % "sbt-native-packager" % "1.8.0")
+addSbtPlugin("com.github.cb372" % "sbt-explicit-dependencies" % "0.2.16")
+addSbtPlugin("org.scoverage" % "sbt-scoverage" % "1.6.0")
```

Comment:
Why did you downgrade it? Are there issues with 1.6.1?

Reply:
Probably missclick

Reply:
Done